node/service/src/lib.rs

```diff
@@ -654,8 +653,8 @@ pub fn new_full<RuntimeApi, Executor>(
 	is_collator: IsCollator,
 	grandpa_pause: Option<(u32, u32)>,
 	jaeger_agent: Option<std::net::SocketAddr>,
-	isolation_strategy: IsolationStrategy,
 	telemetry_worker_handle: Option<TelemetryWorkerHandle>,
+	program_path: Option<std::path::PathBuf>,
```

Comment:
This is the path of the binary itself? Or is it a db-path or somtehing like that?

Reply:
The former one. Do you think the name is confusing?

Reply:
Maybe `exe_path` is less ambiguous and in line with `std::evn::current_exe`?

Reply:
`exe` and `program`  sounds synonymous for me. I suspect the problem may be that it is not immediately obvious what program/executable we are talking about here.